admin_manual/installation/deployment_recommendations.rst

```diff
@@ -3,298 +3,351 @@ Deployment Recommendations
-* Web server: Apache 2.4.
-* Database: MySQL/MariaDB with InnoDB storage engine (MyISAM is not supported, see: :ref:`db-storage-engine-label`)
-* PHP 5.6+.
-
-For Small Workgroups or Departments
------------------------------------
+- Operating system: Linux.
+- Web server: Apache 2.4.
+- Database: MySQL/MariaDB with InnoDB storage engine (MyISAM is not supported, see: :ref:`db-storage-engine-label`)
+- PHP 5.6+.
+- Consider setting up a scale-out deployment, or using `Federated Cloud Sharing`_ to keep individual ownCloud instances to a manageable size.
 
-* Number of users
-   Up to 150 users.
-
-* Storage size
-   100 GB to 10TB.
-
-* High availability level
-   Zero-downtime backups via Btrfs snapshots, component failure leads to 
-   interruption of service. Alternate backup scheme on other filesystems: 
-   nightly backups with service interruption.
+.. note:: Whatever the size of your organization, always keep one thing in mind: 
+   *The amount of data stored in ownCloud will only grow. So plan ahead.*
+
+Scenario 1: Small Workgroups and Departments
+--------------------------------------------
+
+This recommendation applies if you meet the following criteria:
+
+======================= ====================================================
+Option                  Value
+======================= ====================================================
+Number of users         Up to 150 users
+Storage size            100 GB to 10TB
+High availability level Zero-downtime backups via Btrfs snapshots, component 
+                        failure leads to interruption of service. Alternate 
+                        backup scheme on other filesystems: nightly backups 
+                        — with service interruption.
+======================= ====================================================
    
 Recommended System Requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-One machine running the application server, Web server, database server and 
-local storage.
-
+One machine running the application, web, and database server, as well as local storage.
 Authentication via an existing LDAP or Active Directory server.
 
 .. figure:: images/deprecs-1.png
    :alt: Network diagram for small enterprises.
 
-* Components
-   One server with at least 2 CPU cores, 16GB RAM, local storage as needed.
+Components
+~~~~~~~~~~
 
-* Operating system
-   Enterprise-grade Linux distribution with full support from OS vendor. We 
-   recommend Red Hat Enterprise Linux or SUSE Linux Enterprise Server 12.
+One server with at least 2 CPU cores, 16GB RAM, and local storage as needed.
 
-* SSL Configuration
-   The SSL termination is done in Apache. A standard SSL certificate is 
-   needed, installed according to the Apache documentation.
+Operating system
+~~~~~~~~~~~~~~~~
 
-* Load Balancer
-   None. 
+Enterprise-grade Linux distribution with full support from an operating system vendor. 
+We recommend either Red Hat Enterprise Linux and SUSE Linux Enterprise Server 12.
```

Comment:
I always find and/or quite kludgey. (FWIW, I'm not taking a shot at the advice).